mitodl/open-discussions
# Document spam mitigation and modernize docs

#### What are the relevant tickets?
N/A

#### What's this PR do?
This PR is to address a few things:

- Add documentation around our recent spam mitigation efforts
- Get our documentation pages in better shape to be usable and maintainable
  - Update the docs with a more spacious and documentation-appropriate jekyll theme
  - Reorganization and clean up of existing pages
  - Added RFCs to the navigation
    - Added docs on how to write an RFC

#### How should this be manually tested?
Run the docs locally (instructutions in `docs/index.md`), they should look like the screenshot, check the new documentation under "Architecture" in the menu

#### Screenshots (if appropriate)
![Screenshot_2020-08-03 Documentation](https://user-images.githubusercontent.com/28598/89205054-16e9a900-d585-11ea-807d-14ba9128e26f.png)

#### What GIF best describes this PR or how it makes you feel?
![](https://media.giphy.com/media/3ofT5R8ZsrvUPm75vy/giphy.gif)
